.github/CODEOWNERS

```diff
@@ -1790,7 +1790,9 @@ spec/support/vcr_cassettes/bid @department-of-veterans-affairs/Disability-Experi
 spec/support/vcr_cassettes/brd @department-of-veterans-affairs/lighthouse-dash
 spec/support/vcr_cassettes/carma @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/support/vcr_cassettes/caseflow @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-spec/support/vcr_cassettes/charon @department-of-veterans-affairs/lighthouse-banana-peels
+spec/support/vcr_cassettes/charon/400.yml @department-of-veterans-affairs/lighthouse-banana-peels
```

Comment:
Any charon related VCR cassettes should be able to be removed as that validation is no longer occurring.

These were incorrectly associated to lighthosue-banana-peels and should have been lighthouse-pivot codeowners when originally rolled out.

Reply:
removing the "spec/support/vcr_cassettes/charon" alerts a CODEOWNERS entry error to add an entry for each of the files that were deleted and keeping "spec/support/vcr_cassettes/charon" alerts an error to delete it. Adding an entry for each of the files is the only thing that keeps codeowners happy